app/services/filter_service.rb

```diff
@@ -0,0 +1,12 @@
+class FilterService < BaseService
+  def initialize(filters: nil, data_services: nil)
```

Comment:
The nil default makes the parameters optional, so that the class can be instantiated and used without providing values for the filter and search. I'll combine this service with the search and then I'll write a test :)

Reply:
If they're both nil it should return all of the data services, will confirm with a test :)

Reply:
But what's the point of doing that? If you instantiate it with no parameters shouldn't it fail?

Reply:
The functionality has been moved into the Search Service now.

Reply:
I think I see what you mean. Should it instead have validation to check if both are nil, and then if they are then prompt the user to do one or the other?